Volume/Views/MainView/BookmarksList.swift

```diff
@@ -11,6 +11,7 @@ import SwiftUI
 
 struct BookmarksList: View {
     @State private var cancellableQuery: AnyCancellable?
+    @EnvironmentObject private var networkState: NetworkState
```

Comment:
I would group `@State` vars at the top and then `@EnvironmentObject` vars underneath. So:

```
@State private var cancellableQuery: AnyCancellable?
@State private var state: BookmarksListState = .loading
@EnvironmentObject private var networkState: NetworkState
@EnvironmentObject private var userData: UserData
```

Reply:
Sorry that's what I originally thought but then I thought alphabetizing got priority.  I think I get what you mean now though